salt/pkgrepo/ubuntu/init.sls

```diff
@@ -1,3 +1,16 @@
+legacy-saltstack-pkgrepo:
```

Comment:
I don't feel strongly either way. I'm all for explicit over implicit, so if we can make it descriptive that would be good. Maybe something like `disable_legacy_repos: True` or something.